armi/tests/test_lwrInputs.py

```diff
@@ -42,15 +47,35 @@ def tearDownClass(cls):
     def test_loadC5G7(self):
         """
         Load the C5G7 case from input and check basic counts.
+        (Also, check that we are getting warnings when reading the YAML.)
         """
-        o = armi_init(fName=TEST_INPUT_TITLE + ".yaml")
-        b = o.r.core.getFirstBlock(Flags.MOX)
-        # there are 100 of each high, medium, and low MOX pins
-        fuelPinsHigh = b.getComponent(Flags.HIGH | Flags.MOX)
-        self.assertEqual(fuelPinsHigh.getDimension("mult"), 100)
+        with mockRunLogs.BufferLog() as mock:
+            # we should start with a clean slate
+            self.assertEqual("", mock._outputStream)
+            runLog.LOG.startLog("test_loadC5G7")
+            runLog.LOG.setVerbosity(WARNING)
+
+            # ingest the settings file
+            Mode.setMode(Mode.BATCH)
+            o = armi_init(fName=TEST_INPUT_TITLE + ".yaml")
+            b = o.r.core.getFirstBlock(Flags.MOX)
+
+            # test warnings are being logged for malformed isotopics info in the settings file
+            streamVal = mock._outputStream
+            self.assertGreater(streamVal.count("[warn]"), 32, msg=streamVal)
+            self.assertGreater(streamVal.count("custom isotopics"), 32, msg=streamVal)
+            self.assertIn("Uranium Oxide", streamVal, msg=streamVal)
+            self.assertIn("SaturatedWater", streamVal, msg=streamVal)
+            self.assertIn("fakeBad", streamVal, msg=streamVal)
```

Comment:
might make sense to check more than the name, including the fact that it's being ignored?